crates/test_utils/wasm/wasm_workspace/sys_time/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use hdk3::prelude::*;
 
 #[hdk_extern]
-fn sys_time(_: ()) -> ExternResult<SysTimeOutput> {
-    Ok(SysTimeOutput::new(hdk3::prelude::sys_time()?))
+fn sys_time(_: ()) -> ExternResult<core::time::Duration> {
```

Comment:
@steveeJ `HdkResult` is deleted in this PR in favour of `ExternResult`

we cannot remove the extern because `hdk3::prelude::sys_time` is not callable from the hsot